web/src/features/charts/BreakdownChart.tsx

```diff
@@ -110,6 +111,13 @@ function BreakdownChart({
       />
       <Divider />
       <Accordion
+        onClick={() => {
+          trackEvent('Data Sources Clicked', {
```

Comment:
Now we're getting quite a few of these I'm wondering if it would be nice to have an enum to centralized them all in our constant.ts file. What do you think?

Reply:
I love enums so I'm in favour of this. It usually also means the app bundles get smaller once it's used more than 2-3 times.

Reply:
Agreed!